### DISCUSSION TITLE
Download Image Folder

### DISCUSSION THREAD

**Discussion by ntcolussi** [requarks/wiki#7351] [Help / Questions]
Hello!

I would like to know if it is possible for me to download a media/root folder (folders containing all the images of the documents) that is in the Wiki to my local computer.

If possible, how do I do this?

**Comment by fnafox** [comment]
You can use any of the backup/sync functionality to dump content from the wiki including media. Navigate to `Admin > Modules > Storage` and then select the one that works best for you.

**Reply by ntcolussi** [reply]
And where do I find this Modules folder? I accessed Admin but I can't find the rest.

**Reply by fnafox** [reply]
In the wiki administration interface there is a link to Storage preferences in the left side navigation. I'm not sure how to describe it more clearly than that.

**Reply by ntcolussi** [reply]
@fnafox  And can you send me a print so I can get an idea?
